Apps/WebClient/src/ClientApp/src/views/publicVaccineCard.vue

```diff
@@ -283,6 +288,7 @@ export default class PublicVaccineCardView extends Vue {
                                 <b-form-input
                                     id="phn"
                                     v-model="phn"
+                                    v-mask="phnMask"
```

Comment:
I'm not sure if this will work, because in the admin site I installed the v-mask package and registered the VueMaskDirective in main.ts.  I think the client app has the older vue-the-mask library installed instead, which may work differently.